src/components/ChannelLink/ChannelLink.tsx

```diff
@@ -24,9 +25,15 @@ const ChannelLink: React.FC<ChannelLinkProps> = ({
   noLink,
   overrideChannel,
   avatarSize = 'default',
+  onNotFound,
   className,
 }) => {
-  const { channel } = useBasicChannel(id || '', { fetchPolicy: 'cache-first', skip: !id })
+  const { channel } = useBasicChannel(id || '', {
+    fetchPolicy: 'cache-first',
```

Comment:
This is not your change, but I think we could safely remove this, it's redundant. `cache-first` is a default policy.